### ISSUE TITLE
Should bbrun set environment Variable CI=true

### ISSUE THREAD

**Issue by Londane** [mserranom/bbrun#13]
I think Bitbucket use the environment Variable "CI" to indicate that the script is running on a pipeline.
Should bbrun automatically set CI to "true"  ?

**Comment by mserranom** [comment]
I don't think so, this variable is normally used to tell whether we're executing the build scripts in the actual CI environment or in our local machines.

To emulate the CI environment we can always execute `bbrun` with `--env CI=true`

**Comment by airtonix** [comment]
Yes it should. the point of this tool is to simulate bitbucket pipelines.

**Comment by mserranom** [comment]
Reopening for discussion. Will reevaluate based on what other CLI tools are doing (circlecli CLI, travis CLI, etc...)

**Comment by ndastur** [comment]
Just my 2cents. But I think as an emulator of the bitbucket pipeline it should. I for instance use the presence of this variable to set up the testing environment. I agree ```--env CI=true``` could be used but that represents poor discoverability and doesn't seem to do what is in the description
